### PR TITLE
Explore: use GrafanaTheme2

### PR DESCRIPTION
**Pull request by olbo98**
**What this PR does / why we need it**:
This PR is used to refactor files that still use GrafanaTheme to instead use GrafanaTheme2

**Special notes for your reviewer**:
Currently it's only `Logs.tsx` that has been refactored but i can look into other files in explore if you would like

### CODE REVIEW

**Comment by ifrost** [comment]
@olbo98 Could you create an issue for it and add more files? :)

**Review by ifrost — APPROVED** [review]

File: public/app/features/explore/Logs.tsx [hunk 1]
```diff
@@ -432,17 +432,17 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
...
-      border: 1px solid ${theme.colors.border2};
+      background-color: ${theme.colors.background.primary};
+      padding: ${theme.spacing(1)} ${theme.spacing(2)};
+      border-radius: ${theme.shape.borderRadius()};
+      margin: ${theme.spacing(2)} 0 ${theme.spacing(1)};
+      border: 1px solid ${theme.colors.border.medium};
     `,
     flipButton: css`
-      margin: ${theme.spacing.xs} 0 0 ${theme.spacing.sm};
+      margin: ${theme.spacing(0.5)} 0 0 ${theme.spacing(1)};
```
Comment (zoltanbedi) [inline public/app/features/explore/Logs.tsx:442]: nit
```suggestion
      margin: ${theme.spacing(0.5, 0, 0, 1)};
```

File: public/app/features/explore/Logs.tsx [hunk 2]
```diff
@@ -432,17 +432,17 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
...
       flex-wrap: wrap;
-      background-color: ${theme.colors.bg1};
-      padding: ${theme.spacing.sm} ${theme.spacing.md};
-      border-radius: ${theme.border.radius.md};
-      margin: ${theme.spacing.md} 0 ${theme.spacing.sm};
-      border: 1px solid ${theme.colors.border2};
+      background-color: ${theme.colors.background.primary};
+      padding: ${theme.spacing(1)} ${theme.spacing(2)};
+      border-radius: ${theme.shape.borderRadius()};
+      margin: ${theme.spacing(2)} 0 ${theme.spacing(1)};
```
Comment (zoltanbedi) [inline public/app/features/explore/Logs.tsx:438]: nit
```suggestion
      margin: ${theme.spacing(2, 0, 1)};
```

File: public/app/features/explore/Logs.tsx [hunk 3]
```diff
@@ -432,17 +432,17 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       justify-content: space-between;
       align-items: baseline;
       flex-wrap: wrap;
-      background-color: ${theme.colors.bg1};
-      padding: ${theme.spacing.sm} ${theme.spacing.md};
-      border-radius: ${theme.border.radius.md};
-      margin: ${theme.spacing.md} 0 ${theme.spacing.sm};
-      border: 1px solid ${theme.colors.border2};
+      background-color: ${theme.colors.background.primary};
+      padding: ${theme.spacing(1)} ${theme.spacing(2)};
```
Comment (zoltanbedi) [inline public/app/features/explore/Logs.tsx:436]: nit
```suggestion
      padding: ${theme.spacing(1, 2)};
```

**Review by zoltanbedi — APPROVED** [review]
nice!

**Pull request merged**